src/NUnitFramework/tests/Constraints/NUnitEqualityComparerTests.cs

```diff
@@ -22,9 +22,11 @@
 // ***********************************************************************
 
 using System;
+using System.Collections;
 using System.Collections.Generic;
 using System.IO;
 using NUnit.TestUtilities;
+using System.Linq;
```

Comment:
I don't see where you used it below, but you cannot use Linq because it will not compile in the older versions of the framework. That is why the CI builds are failing.